package.json

```diff
@@ -131,6 +131,7 @@
     "lodash-decorators": "4.3.1",
     "mime": "2.4.3",
     "moment": "2.22.2",
+    "moment-duration-format": "^2.3.2",
```

Comment:
I don't think we really need to add a new library to the codebase.  The streaming service already has a function that does this called `formattedDurationSince`.  Look at how we do this for streaming timer.

Reply:
It's actually just a plugin for moment.js

Will use `formattedDurationSince`